src/Groups/matrices/FiniteFormOrthogonalGroup.jl

```diff
@@ -110,7 +110,7 @@ function _normal(G::Union{fmpz_mod_mat, nmod_mat}, p)
   end
 end
 
-function lift(x::AbstractAlgebra.Generic.MatSpaceElem{gfp_fmpz_elem})
+function lift(x::gfp_fmpz_mat)
```

Comment:
Thank you! I didn't realize this method was here. So I guess it makes sense that you fixed it here. I am open to discussing whether this method "makes sense" and thus should be kept (and then perhaps be moved to Nemo?) or whether we should call something else in this code.